packages/pyright-internal/src/common/realFileSystem.ts

```diff
@@ -453,7 +453,7 @@ export class WorkspaceFileWatcherProvider implements FileWatcherProvider, FileWa
         // for a file change. It is event handler's job to filter those out.
         this._fileWatchers.forEach((watcher) => {
             if (watcher.workspacePaths.some((dirPath) => fileUri.pathStartsWith(dirPath))) {
-                watcher.eventHandler(eventType, fileUri);
+                watcher.eventHandler(eventType, fileUri.getFilePath());

```

Comment:
Why do we want to convert uris to strings here?

Reply:
Cause the eventHandler type takes a string now (I was wrong in thinking I could change it). It's external but this code is mimicking it.